bin/assembleCoadd.py

```diff
@@ -21,6 +21,28 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 import lsst.pex.logging as pexLog
-from lsst.pipe.tasks.assembleCoadd import AssembleCoaddTask
+from lsst.pipe.tasks.assembleCoadd import AssembleCoaddTask, SafeClipAssembleCoaddTask
+import sys
+import argparse
 
-AssembleCoaddTask.parseAndRun()
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('--legacyCoadd', action='store_true', default=False)
+parser.add_argument('-h','--help', action='store_true',default=False)
+result, extra = parser.parse_known_args(sys.argv)
+
+if result.help:
+    print('Meta Arguments:\n  --legacyCoadd \t\tAn option to run with the original assembleCoadd '+\
+          'task which will not attempt to do any safe clipping. If run the config option'+\
+	  'config.measurement.plugins["base_PixelFlags"].any = [] must be set\n')
+    SafeClipAssembleCoaddTask.parseAndRun(args=['--help']+extra[1:])
+
+if result.legacyCoadd:
+    legacy_message = '''
+    Remember, If you use the legacy adder you must set
+    config.measurement.plugins['base_PixelFlags'].any = []
+    so that pixel flags will not look for a non existent mask plane
+    '''
+    print(legacy_message)
```

Comment:
I know it makes little difference for one argument, but can you add `from __future__ import print_function` at the top so that when people see the print here they will understand they can use modern syntax.